test/parallel/test-http2-generic-streams-sendfile.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+const common = require('../common');
+if (!common.hasCrypto)
+  common.skip('missing crypto');
+const assert = require('assert');
+const http2 = require('http2');
+const fs = require('fs');
+const MakeDuplexPair = require('../common/duplexpair');
+
+{
+  const testData = '<h1>Hello World</h1>';
```

Comment:
Nit: I think this is unused in this test?

Reply:
Huh. Yup, done!